### DISCUSSION TITLE
An option to Clear Cache

### DISCUSSION THREAD

**Discussion by amadejpapez** [community/community#6689] [Mobile] [answered]
GitHub app is currently taking nearly 1 GB of storage on iOS. Could there be an option to clear cache to maybe solve this?

![image](https://user-images.githubusercontent.com/63184600/138613344-687b93a9-fd10-4f38-8497-719d034fe9e4.jpeg)

**Comment by aconsuegra** [comment, marked as answer]
Hi @amadejpapez 
thanks for getting in touch and about your feedback. I have created an issue to track this internally, I think it makes a lot of sense to add an option to clean up some disc space.

**Reply by thompson-tomo** [reply]
@aconsuegra has there been any movement in that issue as for me the situation is even worse, github app data is consuming 4.5gb on my 64gb ipad. 🤔

![image](https://github.com/user-attachments/assets/ddf9ae0e-eaff-40ce-80bf-fd02d2958496)

**Reply by aconsuegra** [reply]
Hello @thompson-tomo. Yes, we built a feature to clear the cache and free up some disk space. You should see the button in your Profile > Settings (scroll down to the bottom).

**Reply by thompson-tomo** [reply]
Thanks @aconsuegra it appears I was being too eager for the usage to update, can confirm that it is now in the mbs.
